### PR TITLE
Fix extraCommandArgs not being copied

### PR DESCRIPTION
**Pull request by trofim24**
Fix extraCommandArgs not being copied https://github.com/ChadKillingsworth/closure-compiler-npm/issues/18

### CODE REVIEW

File: lib/node/closure-compiler.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ var contribPath = path.dirname(compilerPath) + '/contrib';
  * @param {Array<String>=} extraCommandArgs
  */
 function Compiler(args, extraCommandArgs) {
-  this.commandArguments = extraCommandArgs || [];
+  this.commandArguments = extraCommandArgs.slice() || [];
```
Comment (Dominator008) [inline lib/node/closure-compiler.js]: This won't work if `extraCommandArgs` is `undefined`. I think this line can just be:
```js
this.commandArguments = (extraCommandArgs || []).slice();
```

**Comment by Dominator008** [comment]
Can you edit the commit message title to be clearer? i.e. "Fix extraCommandArgs not being copied". Thanks!

**Comment by trofim24** [comment]
done

File: lib/node/closure-compiler.js (hunk 1 above)
Comment (trofim24) [inline lib/node/closure-compiler.js]: You're right

**Comment by Dominator008** [comment]
Closing this in favor of #20.

**Pull request closed without merging**